### PR TITLE
sample_multi_transcode: enable HEVC encode with temporal layers

### PR DESCRIPTION
**Pull request by jonrecker**
New test suite: smt_hevcd_to_hevce_temporal_layers
Add new switches for temporal layers
Update par file parsing to handle string literals
WA for compiler error on Windows - too many if-else
  clauses in cmd-line parsing causes build error

### CODE REVIEW

File: samples/sample_multi_transcode/src/pipeline_transcode.cpp [hunk 1]
```diff
@@ -2573,6 +2585,38 @@ MFX_IOPATTERN_IN_VIDEO_MEMORY : MFX_IOPATTERN_IN_SYSTEM_MEMORY);
...
 
+    if (pInParams->nAvcTemp)
+    {
+        if (pInParams->EncodeId == MFX_CODEC_HEVC)
+        {
+            int i;
+            for (i = 0; i < 8; i++)
+            {
+                m_AvcTemporalLayers.Layer[i].Scale = pInParams->nAvcTemporalLayers[i];
+                m_AvcTemporalLayers.BaseLayerPID = pInParams->nBaseLayerPID;
```
Comment (mgonchar) [inline samples/sample_multi_transcode/src/pipeline_transcode.cpp]: why this is in the loop?

File: samples/sample_multi_transcode/src/pipeline_transcode.cpp [hunk 2]
```diff
@@ -2573,6 +2585,38 @@ MFX_IOPATTERN_IN_VIDEO_MEMORY : MFX_IOPATTERN_IN_SYSTEM_MEMORY);
 #endif
     }
 
+    if (pInParams->nAvcTemp)
+    {
+        if (pInParams->EncodeId == MFX_CODEC_HEVC)
+        {
+            int i;
```
Comment (mgonchar) [inline samples/sample_multi_transcode/src/pipeline_transcode.cpp]: better to move it in for loop ```for (int i = 0; ...)```

File: samples/sample_multi_transcode/src/pipeline_transcode.cpp (hunk 1 above)
Comment (jonrecker) [reply samples/sample_multi_transcode/src/pipeline_transcode.cpp]: Fixed. Thanks.
Updated commit:  3f5bc4f

File: samples/sample_multi_transcode/src/pipeline_transcode.cpp (hunk 2 above)
Comment (jonrecker) [reply samples/sample_multi_transcode/src/pipeline_transcode.cpp]: Fixed. Thanks.
Updated commit:  3f5bc4f

**Pull request merged**